updated openemr docker documentation

bradymiller opened the pull request.

bradymiller commented:
@TheToolbox @jesdynf ,
Just some openemr documentation updates.

TheToolbox commented:
LGTM

jesdynf commented on docker-images/openemr/5.0.0/README.md:
```diff
@@ -48,7 +49,12 @@ volumes:
+- Optional settings for auto installation are:
+    - Setting db parameters `MYSQL_USER`, `MYSQL_PASS`, `MYSQL_DATABASE`
+    - Setting openemr parameters `OE_USER`, `OE_PASS`
+- Can override auto installation and force manual installation if wish by setting `MANUAL_SETUP` environment setting to 'yes'
```
May move back to conversational English for this section in general; 'if wish' in specific is never correct.

bradymiller commented:
@jesdynf @TheToolbox 
Thanks for the input. I took a stab at addressing the issues and also included instructions for the new ssl feature.
thanks, -brady
